.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'es5',
+  printWidth: 100,
+};
```

Comment:
*Although* the core idea of prettier is to prevent bikeshedding, I have to do it 😝
We have this in our common `.prettierrx.yml`:

```yaml
arrowParens: always # It's convenient that you don't have to add them by hand when you need to add more args or a type
singleQuote: true   # Enabled to align with EmberJS Prettier config
printWidth: 120     # Per team's agreement
trailingComma: es5  # Trailing commas makes editing easier, diffs cleaner. `es5` option aligns with EmberJS Prettier config
```

What do you think? Re: `printWidth` I think everybody has a wide enough monitor to support this. Things often got harder to read from our experience when prettier forced line wraps to eagerly...

Reply:
Prettier changed the default value for arrow function parens to always include them in v2.

Only meaningful difference is print width if I didn't missed something. Prettier explicitly recommends against a print width of 120:

> With the print width set to 120, prettier may produce overly compact, or otherwise undesirable code.

See prettier docs for more details: https://prettier.io/docs/en/options.html#arrow-function-parentheses

I try to avoid having a personal opinion on that topics at all. 😜

Reply:
> Prettier explicitly recommends against a print width of 120:

But also against one of 100! :wink:

IIRC one thing that was for example hard to read was function invocations/definitions with a couple of arguments, with longer (speaking) names. When breaking this up into one line per argument, this became difficult to read. As when you are skimming through code to see what's happening, you want to see which functions are called, but not necessarily with which arguments (at this point). But having the function name and all arguments all split up into separate lines (with the arguments just having those 2 more spaces of indentation) made it hard to immediately see what's a function name and what's an argument. Just one example...

So when you don't have strong feelings, I would suggest to use that width of 120. 🙂

Reply:
Changed `printWidth` to `120` in 822b5bb3af41da2d2df2446f85fc1c4884e523f4.

Reply:
I also removed `trailingComma: 'es5'` in 6a717e3aebe483b9183137b279fbf327bec18d9c cause that's the default value for pretter@^2.0. Only had it included due to not diverge from configuration used in Ember.